packages/react-compose/src/resolveSlotProps.test.ts

```diff
@@ -0,0 +1,27 @@
+import { resolveSlotProps } from './resolveSlotProps';
+
+describe('resolveSlotProps', () => {
```

Comment:
this can use more tests too, perhaps after a round of review

Reply:
Already in progress.